src/compiler/commandLineParser.ts

```diff
@@ -1160,7 +1160,13 @@ namespace ts {
                         }
                     }
                     else {
-                        errors.push(createCompilerDiagnostic(unknownOptionDiagnostic, s));
+                        const possibleOption = getSpellingSuggestion(s, optionDeclarations, opt => `--${opt.name}`);

```

Comment:
You probably also want this when parsing tsconfig options.. Eg. https://github.com/microsoft/TypeScript/blob/37820defaeeae460b508b231ada5dd6c6337be17/src/compiler/commandLineParser.ts#L1400

Reply:
Also note the tsconfig can be parsed by creating sourceFile to report error at location or by just using json apis.. So you would need to handle https://github.com/microsoft/TypeScript/blob/37820defaeeae460b508b231ada5dd6c6337be17/src/compiler/commandLineParser.ts#L2479 likes as well

Reply:
Thanks, cool, will take a look!